### PR TITLE
docs: example to test output value

### PR DESCRIPTION
**Pull request by rajatagarwal-ibm**
### Description

Example to test the output value.

### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,21 @@ func TestRunBasic(t *testing.T) {
 ```
 ___
 
+### Example to check an output value in the test
+
```
Comment (daniel-butler-irl) [inline README.md:92]: I want to add some context here
> After the test completes and teardown occurs, the state file will no longer contain the outputs. To handle this the last test to execute stores its outputs in `LastTestTerraformOutputs`. There is also a helper function to validate that the outputs exist called `ValidateTerraformOutputs` this returns a list of output keys that are missing and an error message with details of the missing keys.

File: README.md [hunk 2]
```diff
@@ -88,6 +88,23 @@ func TestRunBasic(t *testing.T) {
...
+The following example checks if the output exists and contains a certain value.
+
+After the test completes and teardown occurs, the state file will no longer contain the outputs. To handle this the last test to execute stores its outputs in `LastTestTerraformOutputs`. There is also a helper function to validate that the outputs exist called `ValidateTerraformOutputs` this returns a list of output keys that are missing and an error message with details of the missing keys.
```
Comment (SirSpidey) [inline README.md:95]:
```suggestion
After the test completes and teardown occurs, the state file no longer contains the outputs. To handle this situation, the last test to execute stores its outputs in `LastTestTerraformOutputs`. Use the helper function called `ValidateTerraformOutputs` to validate that the outputs exist. The function returns a list of output keys that are missing and an error message with details.
```

**Review by SirSpidey — CHANGES_REQUESTED** [review]
minor editorial suggestions.

**Review by daniel-butler-irl — APPROVED** [review]

**Review by SirSpidey — APPROVED** [review]
👍

**Pull request merged**